### PR TITLE
Second revision of Gear' whitepaper

### PR DESCRIPTION
**Pull request by AndrePanin**
completely rewritten

### CODE REVIEW

File: docs/capabilities/_category_.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "label": "Capabilities & Use Cases",
+    "position": 5
+  }
```
Comment (shamilsan) [inline docs/capabilities/_category_.json:4]:
```suggestion
}

```

File: docs/capabilities/use-cases/onboarding.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Onboarding
+sidebar_position: 1
+---
+
+The key to the success and broad adoption of Gear technology lies in its ease of use and fast onboarding process, especially for program developers.
+
+To facilitate this,a comprehensive knowledge hub, the **[Gear Wiki](https://wiki.gear-tech.io/)**, is available. This resource includes detailed information about Gear's components, Rust libraries, how to build, run, and interact with the Node, and how to write and test programs.
```
Comment (shamilsan) [inline docs/capabilities/use-cases/onboarding.md:8]:
```suggestion
To facilitate this, a comprehensive knowledge hub, the **[Gear Wiki](https://wiki.gear-tech.io/)**, is available. This resource includes detailed information about Gear's components, Rust libraries, how to build, run, and interact with the Node, and how to write and test programs.
```

File: docs/components/_category_.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+    "label": "Interface and components",
+    "position": 6
+  }
```
Comment (shamilsan) [inline docs/components/_category_.json:4]:
```suggestion
}

```

File: docs/capabilities/use-cases/_category_.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+    "label": "Applications & Use Cases",
+    "position": 4
+  }
```
Comment (shamilsan) [inline docs/capabilities/use-cases/_category_.json:4]:
```suggestion
}

```

File: docs/components/gearlib.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: Gear Library
+sidebar_position: 2
+---
+
+The available custom developer-friendly library, empowering dApp developers to execute arbitrary logic programs, including unit and integration tests, using `std` functions. This library streamlines the implementation of asynchronous logic through Gear`s messaging paradigm.
```
Comment (shamilsan) [inline docs/components/gearlib.md:6]: Where is a predicate here?

File: docs/components/node.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+To run a Gear node, anyone can compile it on their own or download the prebuilt binary (nightly build) which is the same from a functional standpoint.
+
+### Personal Node
+
+Personal nodes means anyone’s device with Gear’s node up and running, connected either to the Gear test network, operated in a standalone dev net mode or as a part of a multi-node local testnet. There are no specific hardware requirements for personal nodes other than having an SSD.
+
+### Public Nodes
+
+Public nodes refer to devices that act as validators in a Gear public network. The nodes must run on a server that has a public IP address. Hardware requirements is the subject of evaluation. Nodes participating in the public network must share their real-time data for health checks (continuous telemetry collection).
```
Comment (shamilsan) [inline docs/components/node.md:16]: Why is the public IP for? Usually, we don't need it for a P2P connection.

File: docs/components/gearlib.md (hunk 5 above)
Comment (AndrePanin) [reply docs/components/gearlib.md]: updated

File: docs/components/node.md (hunk 6 above)
Comment (AndrePanin) [reply docs/components/node.md]: updated

**Review by peter-cmw — APPROVED** [review]

**Pull request merged**